tasks/st1105_primary_server_port_check.rb

```diff
@@ -32,8 +32,12 @@ def port_test(dest, port)
   true
 end
 
+deprecation_msg = "This task is deprecated and will be removed in a future release. Please see this module's README for more information."
```

Comment:
this is a string, the rest of the return array is a json hash, i would put another hash in there